SS14.Labeller/Database/DatabaseMigration.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentMigrator.Runner;
+
+namespace SS14.Labeller.Database;
+
+public sealed class DatabaseMigration
+{
+    public static void MigrateDatabase(IServiceProvider sp)
+    {
+        using (var scope = sp.CreateScope())
```

Comment:
can't you inline the using statement?
```suggestion
        using var scope = sp.CreateScope()
```